tasks/79_calculate_primoral_of_a_number/calculate_primoral_of_a_number.js

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable for-direction */
+//  I modified slightly my code used in previous exercises.
+
+export const numPrimorial = ( num ) => {
+  let primesArray = [];
+  for ( let i = 2; i > 0; i ++) {
+    if ( primesArray.length + 1 <= num ) {
+      getPrime( i, primesArray );
+    } else {
+      break;
+    }
+  }
+  return primesArray.reduce(( prev, next ) => prev * next, 1);
```

Comment:
you can omit second argument to reduce function and it will take first element of array for first iteration as prev